lib/resources/r4/appointment.yaml

```diff
@@ -103,7 +96,6 @@ fields:
 
 - name: serviceType
   required: 'Yes'
```

Comment:
previous cardinality 0..1 doesn't match with required field, can you double check?

Reply:
[double checked](https://github.com/cerner/fhir.cerner.com/pull/374/commits/6fb0b4a5a1bdcd0b07e086f542d44ded83b12430?file-filters%5B%5D=.yaml#diff-fbb215a73c336ca1ac34d91de2fb411eL98)